File: SDKs/googleapis.com/realtimebidding/v1/python/src/sdk/models/shared/creativedimensions.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class CreativeDimensions:
    r"""CreativeDimensions
    The dimensions of a creative. This applies to only HTML and Native creatives.
    """
    
    height: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('height') }})
    width: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('width') }})
    

File: SDKs/googleapis.com/realtimebidding/v1/java/lib/src/main/java/openapisdk/models/shared/CreativeDimensions.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * CreativeDimensions
 * The dimensions of a creative. This applies to only HTML and Native creatives.
**/
public class CreativeDimensions {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("height")
    public String height;
    public CreativeDimensions withHeight(String height) {
        this.height = height;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("width")
    public String width;
    public CreativeDimensions withWidth(String width) {
        this.width = width;
        return this;
    }
}